app/config/locale/translations/en.json

```diff
@@ -11,13 +11,20 @@
     "emails.verification.signature": "{{project}} team",
     "emails.magicSession.subject": "{{project}} Login",
     "emails.magicSession.hello": "Hello,",
-    "emails.magicSession.optionButton": "Click the button below to securely sign in to your {{project}} account. It will expire in 1 hour.",
+    "emails.magicSession.optionButton": "Click the button below to securely sign in to your {{projectBold}} account. It will expire in 1 hour.",
     "emails.magicSession.buttonText": "Sign in to {{project}}",
     "emails.magicSession.optionUrl": "If you are unable to sign in using the button above, please visit the following link:",
     "emails.magicSession.clientInfo": "This sign in was requested using {{agentClient}} on {{agentDevice}} {{agentOs}}. If you didn't request the sign in, you can safely ignore this email.",
     "emails.magicSession.securityPhrase": "Security phrase for this email is {{phrase}}. You can trust this email if this phrase matches the phrase shown during sign in.",
     "emails.magicSession.thanks": "Thanks,",
     "emails.magicSession.signature": "{{project}} team",
+    "emails.otpSession.subject": "{{project}} Login",
+    "emails.otpSession.hello": "Hello,",
+    "emails.otpSession.description": "Enter the following verification code when prompted to securely sign in to your {{projectBold}} account. It will expire in 15 minutes.",
```

Comment:
```suggestion
    "emails.otpSession.description": "Enter the following verification code when prompted to securely sign in to your {{projectBold}} account. This code will expire in 15 minutes.",
```

Reply:
We have no real pattern here for adding new keys and the naming is going wild.. lets just stick to using {{project}} until there is a better solution

Reply:
We already used projectBold in like 3 emails in different PRs. Chen really wanted to make it bold.